src/platform/packages/shared/kbn-discover-utils/src/data_types/logs/constants.ts

```diff
@@ -60,6 +59,16 @@ export const FILTER_OUT_FIELDS_PREFIXES_FOR_CONTENT = [
   'orchestrator.',
   'log.',
   'service.',
+  '_id',
+  '_index',
+  '_source',
+  '_size',
+  '_doc_count',
+  '_field_names',
+  '_ignored',
+  '_routing',
+  '_meta',
+  '_tier',
```

Comment:
Hi @rStelmach,

Looks like `FILTER_OUT_FIELDS_PREFIXES_FOR_CONTENT` is used for checking whether a field name starts with one of the array items or not. Since we would like to exclude only specific fields, extending this list would not work correctly and we would need (for example) to define another array and extend the logic in `isFieldAllowed` util accordingly.

Reply:
Hello @jughosta, I changed the logic for filtering, let me know if that's what you had in mind.

Reply:
Thanks! Now we can remove the new fields from `FILTER_OUT_FIELDS_PREFIXES_FOR_CONTENT`.

Reply:
Oh you are right, forgot to change it 😄 .